Forge/build.gradle

```diff
@@ -63,23 +109,43 @@ minecraft {
     }
 }
 
-sourceSets.main.resources.srcDir 'src/generated/resources'
+//createMcpToSrg.finalizedBy('genIntellijRuns')
```

Comment:
Automatically adding Forge run configurations to Intellij once the proect loads. Commented doe to being not a good way to do so.

Reply:
This may lead to the runs being inadvertently regenerated when a user doesn't want to generate them.

Also while this template currently doesn't work in eclipse, I do hope that someday it will, in which case you wouldn't want to generate the intelliJ runs